src/main/java/com/gl/donate_receive/donate_receive/model/User.java

```diff
@@ -0,0 +1,21 @@
+package com.gl.donate_receive.donate_receive.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Data
+@Table(name="users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID userId;
+    private String role;
+    private String login;
```

Comment:
As a general rule - less code - less opportunities for a bug. This, however is a different thing. I'd recommend to add all the annotations for every field. 

Login in this case is a unique field that must not be null and may contain up to 320 characters:
![image](https://user-images.githubusercontent.com/108729712/177326385-689b9ca1-a9b1-4c4a-b5ef-c0ef6814cedc.png)